### PR TITLE
Write and read data 

### PR DESCRIPTION
**Pull request by vanBeelenReka**
- I used CSV files
- there are a difference how I use data and how I store them in the csv files

### CODE REVIEW

File: wordcards.py [hunk 1]
```diff
@@ -26,11 +25,43 @@ def ask_back_words(cards):
...
+    clean_cards = list(cards.items())
+    return clean_cards
+
+
+def write_words_to_file(cards, file_name):
+    clean_cards = make_cards_clean_to_write_in_a_file(cards)
+    with open(file_name, "w") as file:
+
+        for item in clean_cards:
+            file.write(str(item[0] + "-" + item[1] + ";"))
```
Comment (lwvb) [inline wordcards.py]: This step where the data format is changed to something that can be written to a file is called serialize. Usually you have a serialize and a deserialize function for it. 
```suggestion
    file.write(serializeCards(clean_cards))

def serializeCard(card):
    return str(card[0] + "-" + card[1] + ";")
def serializeCards(cards):
    serializedCards = map(serializeCard, cards)
    return ''.join(serializedCards)
    

```

File: wordcards.py [hunk 2]
```diff
@@ -26,11 +25,43 @@ def ask_back_words(cards):
...
+
+
+def transform_exported_data(lines):
+    word_cards_dic = {}
+    for item in lines:
+        word_groups = item.split(";")
+        for groups_items in word_groups[0:-1]:
+            word_pairs = groups_items.split("-")
```
Comment (lwvb) [inline wordcards.py:53]: You sometimes might need a - in a word. The python csv module uses ; and newlines as separators: https://docs.python.org/3/library/csv.html

**Review by lwvb — APPROVED** [review]
Well written code and it seems to work fine. I added some suggestions for improvements but it is fine without it as well

File: wordcards.py (hunk 2 above)
Comment (vanBeelenReka) [reply wordcards.py:53]: That is true, but  I am already using ; between two word pair. I should put just one word pair in a line? So newline separator could be the separator between two word pairs

**Pull request merged**